### ISSUE TITLE
The tint is fixed when reusing the icon used in "ExpandableBootomBar".

### ISSUE THREAD

**Issue by crowrish** [st235/ExpandableBottomBar#32]
Here is my `menu/bottom.xml`
```
<?xml version="1.0" encoding="utf-8"?>
<menu
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:android="http://schemas.android.com/apk/res/android">

    <item
        android:id="@+id/bottom_home"
        android:title="@string/menu_home"
        app:exb_color="#000000"
        app:exb_icon="@drawable/round_home_black_24" />
    ...
</menu>
```

and here is my `activity_main.xml`

```
...
<github.com.st235.lib_expandablebottombar.ExpandableBottomBar
            android:id="@+id/bottom_bar"
            android:layout_width="match_parent"
            android:layout_height="80dp"
            android:layout_margin="20dp"
            app:layout_behavior="github.com.st235.lib_expandablebottombar.behavior.ExpandableBottomBarScrollableBehavior"
            app:layout_constraintBottom_toBottomOf="parent"
            app:exb_backgroundCornerRadius="10dp"
            app:exb_itemInactiveColor="#ccc"
            app:exb_items="@menu/bottom"
            />
...
```

If I try to use the `@drawable/round_home_black_24` drawable icon in another view, the tint of the drawable is also applied to `#ccc` in the other view.

here is image.
![KakaoTalk_20200521_153513872](https://user-images.githubusercontent.com/60930468/82530726-e8108900-9b78-11ea-8598-45dac480a4d2.jpg)

The original color of the drawable is black, and I haven't set the tint property.

**Comment by st235** [comment]
Hi 👋 
Yeah, it looks like a bug. I will take a look the problem deeper soon, but now I can only suggest you to use different resources for menu items and bottom bar as a temporary solution.

I will keep you updated about the progress

**Comment by st235** [comment]
@crowrish, the behaviour was fixed, please, update your version to 1.1.1

**Comment by crowrish** [comment]
I applied version `1.1.5` migration, and this update works fine.
Thank you for your help. 👍
